app/components/find/courses/about_schools_component/view.html.erb

```diff
@@ -23,7 +22,7 @@
 
     <% if course.site_statuses.map(&:site).uniq.many? %>
       <h3 class="govuk-heading-m">Locations</h3>
-      <p class="govuk-body">You can select one of the following locations when you apply.</p>
+      <p class="govuk-body">You can select one of the following locations when you apply to indicate a preferred placement.</p>
```

Comment:
You can select one of the following locations to indicate a preferred placement when you apply.